pkg/catalog/catalog_test.go

```diff
@@ -294,7 +310,7 @@ func TestGetDevfileIndex(t *testing.T) {
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			got, err := GetDevfileIndex(tt.devfileIndexLink)
+			got, err := getDevfileIndexEntriesFrom(tt.registryName, tt.registryURL)
```

Comment:
This test was for testing GetDevfileIndex() but now it's testing getDevfileIndexEntriesFrom(). Since getDevfileIndexEntriesFrom() is unused in the code implementation and internally calls `getDevfileIndexEntries()` let's rename the test and test getDevfileIndexEntries instead?

Reply:
Makes sense indeed, I didn't properly look at the context. This is fixed now.